force-app/main/default/classes/SOQL.cls

```diff
@@ -201,7 +293,13 @@ public with sharing class SOQL {
      * @return A list of Cases
      */
     public static List<Case> getCasesWithStatus() {
-        return null; // Replace null with the variable you used to store the result
+            List<Case> openCaseswithStatus = [SELECT Id, Status, CreatedDate
+                                            FROM Case
+                                            WHERE Status = 'Open'
```

Comment:
Here **Open** works but only because the test method uses it. By default, the Status picklist values on a Case are **On Hold**, **Escalated**, **Closed**, and **New**. Based on those, what might be a better way to ensure only 'open' cases are returned?